zhazam/Modules/Loading/LoadingViewController.swift

```diff
@@ -0,0 +1,39 @@
+//
+//  LoadingViewController.swift
+//  zhazam
+//
+//  Created by Nurbek Ismagulov on 8/20/19.
+//  Copyright © 2019 Nurbek Ismagulov. All rights reserved.
+//
+
+import UIKit
+
+final class LoadingViewController: UIViewController {
+    enum LocalConstants {
+        static let loadedPercentage: Int = 10
+    }
+    
+    @IBOutlet var percentageLabel: UILabel!
```

Comment:
I think you accidentally removed `private` access modifier when you was getting rid of unnecessary `weak`. Need to get it back certainly as we don't want to expose our UI internals to outside world, right? :)